security/suricata/files/suricata.in

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-# $FreeBSD$
+# $FreeBSD: head/security/suricata/files/suricata.in 454649 2017-11-21 18:55:28Z zeising $
```

Comment:
Please don't add svn macro expansion here

Reply:
Removed. Another instance that came over from FreeBSD-ports upstream.